### ISSUE TITLE
typing `while foo <` autocompletes to `while foo <>`

### ISSUE THREAD

**Issue by trafnar** [imba/vscode-imba#108]
if you want to type `while foo < bar` it autocompletes to `while foo <>`  forcing you to delete the extraneous `>`

**Comment by somebee** [comment]
There's really no way about it in vscode atm, but the first result of the completion should be a blank item that completes by typing space? So that typing `<` and then ` ` should erase the closing `>`? If it does not happen for you, see if you have `editor.acceptSuggestionOnCommitCharacter` toggled off in vscode settings. Maybe see if changing `editor.suggestSelection` to `first` also helps for this.

**Comment by trafnar** [comment]
I tried changing that setting with the same result, I actually immediately get the closing `>` upon typing the opening `<` without having to press space or anything. Same thing doesn't happen with typescript typing `while(foo < bar)`

![Screen Recording (Code)](https://user-images.githubusercontent.com/6732/131134182-20ab6427-2e7c-40c8-995a-aa74568f99af.gif)

**Comment by somebee** [comment]
It is very weird that you don't get a list of completions there. Do you never get a list of tags when you write `<`?

![image](https://user-images.githubusercontent.com/8467/131982244-430e74a1-7e3b-4078-8570-6e4c5b4af747.png)

**Comment by somebee** [comment]
Do you have `editor.suggestOnTriggerCharacters` turned on? If not, could you try that?

**Comment by somebee** [comment]
Do you still have this problem @trafnar? Did you check the setting I mentioned? If you're not seeing the same as my screenshot above there is something very wrong with completions on your setup so I'd love to figure out why :)

**Comment by trafnar** [comment]
Yes it does still happen. I tried setting `editor.suggestOnTriggerCharacters` to true and it still happens.

**Comment by somebee** [comment]
Does that mean that you do _not_ get any suggestions usually when you type in `<` in an imba file? Another potential reason: Can you set "editor.suggestSelection" to "first"? If that is the culprit I'll try to find ways around it

**Comment by trafnar** [comment]
ok, when I turned off `editor.acceptSuggestionOnCommitCharacter` it fixed the issue. And I see now that you asked me to do that in your first comment but I missed it.. my bad 😭

**Comment by trafnar** [comment]
This issue actually happens in the scimba editor too!
